lib/src/bloc/movies_bloc.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:async';
+import '../models/item_model.dart';
+import '../resources/movies_repository.dart';
+
+class MoviesBloc {
```

Comment:
For the future it would be good to make Bloc conform to a generic interface, remember what we did on the Dice repository?